__tests__/index.test.js

```diff
@@ -107,3 +107,62 @@ describe("GET /api/articles/:article_id", () => {
     });
   });
 });
+describe("GET /api/articles", () => {
+  it("GET 200: should serve an array of all articles, where each has the following attributes: author, title, article_id, topic, created_at, votes, article_img_url, comment_count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const {articles} = body
+        console.log(articles[2]);
+        const actual = articles[2];
```

Comment:
You should check the length of the returned array here, and loop through the whole array checking the general shape of all the returned objects (i.e. what keys do they have, what datatype are their values). 
This makes sure all the articles are the correct shape, not just one of them